### ISSUE TITLE
Does not have "Image Scout" Selection in Directus 9.5.2

### ISSUE THREAD

**Issue by nathanhere** [resauce-dev/directus-image-scout#16]
First time Image Scout extension user as of Directus 9.5.2, however it doesn't seem like "Image Scout" is even a selection in the Interface section. Just the default selections are present. Deploying directus using docker compose.

*The extension successfully loaded:*
```bash
directus  | 22:52:52 ✨ Loaded extensions: resauce-image-scout, resauce-image-scout
```
*The Interface options for column type File:*

![Screen Shot 2022-03-01 at 4 50 00 PM](https://user-images.githubusercontent.com/1151667/156262153-93629353-7202-4494-bcb4-b9ee3e2ce906.png)

**Comment by shealavington** [comment]
Hey! Thanks for this update, I'll take a look when I get a moment.

Directus is growing so quickly, unfortunately I've not had time to keep up with the pace of development that's going on there.

I'll let you know when I've got it fixed 👍

**Comment by devTheodul** [comment]
Hey, I just experienced this same issue. There's no Image Scout selection in the interface section.

Any updates on this?

**Comment by shealavington** [comment]
> Hey, I just experienced this same issue. There's no Image Scout selection in the interface section.
> 
> Any updates on this?

Hi, Sorry, I've been extremely busy, I will try to have a look over the next few days, unfortunately, I've been unable to keep up to date with what changes have been made to Directus recently, therefore I'll need to try and find the cause of what's happened.

I'll try to update if I work something out, thanks!

**Comment by shealavington** [comment]
I wanted to give you both an update (@devTheodul @nathanhere), I found a burst of energy and I've begun working on this today. I'm making good progress. With the changes to Directus over the last year, there's a large amount to fix. 

I've got the UI working again, I'm currently looking at getting the imports working as they've stopped, and I'm not sure why it's not suggesting it under the Files interface, however, I've found that doing `Input -> UUID -> Interface: Image Scout` seems to semi-work, but that's less than ideal.

I'll keep this thread updated with progress.
